src/App.js

```diff
@@ -331,433 +266,258 @@ class FeatureLoader extends React.Component<Props, State> {
   }
 
 
-  updateViewportSizeState() {
-    this.setState({ isOnSmallViewport: isOnSmallViewport() });
-  }
-
-
-  accessibilityFilter(): YesNoLimitedUnknown[] {
-    const allowedStatuses = yesNoLimitedUnknownArray;
-    if (!this.state.status) return [].concat(allowedStatuses);
-    const result = this.state.status
-      .split(/\./)
-      .filter(s => includes(allowedStatuses, s));
-    return ((result: any): YesNoLimitedUnknown[]);
-  }
-
-
-  toiletFilter(): YesNoUnknown[] {
-    const allowedStatuses = yesNoUnknownArray;
-    if (!this.state.toilet) return [].concat(allowedStatuses);
-    const result = this.state.toilet
-      .split(/\./)
-      .filter(s => includes(allowedStatuses, s));
-    return ((result: any): YesNoUnknown[]);
-  }
-
-
-  fetchFeature(featureId: ?string): void {
-    if (!featureId) {
-      this.setState({ featureId: null, feature: null, lat: null, lon: null, zoom: null });
-      return;
-    }
-    this.setState({ fetching: true, featureId });
+  fetchFeature(featureId: string): void {
```

Comment:
This does not need to check for a `fetching` flag anymore – it's should only be called when actually needed.